src/main/java/com/lob/model/Letter.java

```diff
@@ -364,8 +413,12 @@ public RequestBuilder setAddressPlacement(String addressPlacement) {
             return this;
         }
 
-        public RequestBuilder setReturnEnvelope(boolean returnEnvelope) {
-            params.put("return_envelope", returnEnvelope);
+        public RequestBuilder setReturnEnvelope(String alias) {

```

Comment:
Is `alias` the correct field? Or should this be `id` per [docs](https://docs.lob.com/#operation/letter_create)